app/_locales/en/messages.json

```
@@ -44,6 +44,24 @@
   "QRHardwareWalletSteps2Description": {
     "message": "Ngrave (Coming Soon)"
   },
+  "SIWEDomainInvalid": {
+    "message": "The website you are attempting to sign-in to ($1) does not match the domain in the sign-in request. Proceed with caution."
```

```suggestion
    "message": "The website you are attempting to sign in to ($1) does not match the domain in the sign-in request. Proceed with caution."
```

nit: I believe we want to use "sign in" instead of "sign-in" in the first instance here to be consistent with the `SIWEWarningBody` message and because we are using it as a verb here. ref: https://english.stackexchange.com/questions/49850/sign-in-signin-or-sign-in